dream/test/test.t

```diff
@@ -37,6 +39,10 @@
   OK: User info for "username_via_path"
   ## GET /Signal?level=2
   OK: Signal: 2
+  ## GET /List_users?user_ids=u1&user_ids=u2
+  OK: User ids = [u1, u2]
+  ## GET /List_users
+  OK: User ids = []
```

Comment:
Omitting the param gives an empty list. I believe this is fine, one can always build a `Non_empty_list` or so